core/android_make.go

```diff
@@ -140,14 +141,14 @@ func specifyCompilerStandard(varname string, flags []string) string {
 	return line
 }
 
-func (m *library) GenerateBuildAction(binType int, ctx blueprint.ModuleContext) {
+func (m *library) GenerateBuildAction(sb *strings.Builder, binType int, ctx blueprint.ModuleContext) {
```

Comment:
And in keeping with my comment on line 76, this function (and the other helpers, e.g. `declare*()`) are already correct in having `sb`, the "destination", as the first argument.